Sources/_StringProcessing/PrintAsPattern.swift

```diff
@@ -108,8 +108,8 @@ extension PrettyPrinter {
       print("/* TODO: conditional */")
 
     case let .quantification(amount, kind, child):
-      let amount = amount._patternBase
-      let kind = kind._patternBase
+      let amount = amount.ast._patternBase
+      let kind = (kind.ast ?? .eager)._patternBase
```

Comment:
Does this depend on options?

Reply:
It does in two ways:
- If this quantifier was written in the DSL, it can have a default value that gets drawn from the current options
- If this quantifier was written in syntax, `.eager` and `.reluctant` can get flipped, depending on the options (the AST is just the syntax)

We defer both of those effects until byte code generation — we'd need to maintain option state in a similar way here.